templates/gem/<%= gem_name %>.gemspec

```diff
@@ -0,0 +1,33 @@
+=begin
+Copyright <%= copyright %>
+
+See the file LICENSE for copying permission.
+=end
+
+# -*- encoding: utf-8 -*-
```

Comment:
there is no restriction for ruby version, the main gem should work down to 1.8.7 ... the tests still run fine on 1.9.3